frontend/iframe/viewmodels/SessionViewModel.ts

```diff
@@ -3,8 +3,11 @@ import { RoomViewModel } from "./RoomViewModel";
 import { SettingsViewModel } from "./SettingsViewModel";
 
 export class SessionViewModel extends BaseSessionViewModel {
+    private readonly _singleRoomId: boolean;
```

Comment:
For a boolean, `_singleRoom` or `_singleRoomMode` might be a better name?

Reply:
The type was wrong, should be `string | undefined`. Fixed.

Reply:
Also, good catch 👍